src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipelineb/Launcher.java

```diff
@@ -0,0 +1,45 @@
+package fr.inria.spirals.repairnator.pipelineb;
+
+import fr.inria.spirals.repairnator.pipeline.LauncherAPI;
+import fr.inria.spirals.repairnator.config.RepairnatorConfig;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+import fr.inria.spirals.repairnator.notifier.PatchNotifier;
+
+import com.martiansoftware.jsap.JSAP;
+
+public class Launcher implements LauncherAPI{
```

Comment:
we should not create a new package.


why do we need this class? remove?

Reply:
It's the new Launcher . I changed its name to `BranchLauncher` and it's now in the same package :) .